.travis.yml

```diff
@@ -37,16 +35,15 @@ deploy:
     on:
       tags: true
       condition: $TRAVIS_TAG =~ $PROD_TAG_REGEXP
-  # deploy tags matching v#.#.#  to NPM Registry
-  - provider: npm
-    email: "$NPM_USER_EMAIL_ADDRESS"
-    api_key:
-      secure: "RKsKsLHCiNnJwgvZzU8bpSwYCe1Eq2c5xhotB9K7tXv3baPfbWu+hwDqmWnRk2LGEBYmPK1Z/omhAfPX27CFwUtFSr664td9uNGOyQBzac9K1kLeUwCJqgDAhueI8rmwvivMnjqsiGJ49nsUVGumydqz+qKNDBuk8PfNgeJUtCDWLc2NNQXWJk2cNf4bCo6BaApZpSRjpO453EcqDllbUCXYo4KU9rjjmcTiHJty/poUly5yabs0SZzD2wrsf5V6dtzFbI2gt1/GtlaG0xD9+TQMl4m0yE5QXl/lvU4B+1t4jb4BBHyV3iAt1TCysofxb/88zjGFpv+0LoY2aaUnGczaAA0wMNhkk6fsSMJ4Z6lYZksR9pAWRBjnPyzGf1yf3Aq9IlSWeQn5BgSeyd1DIduHQk7K/b6wtlT2zWKqw2zF1jt9KHB6WQ7O0gQl6PqL9uCZTCwgSNxU1QxZO3uuFZI5Vd8pjvZs/mS+Jxkq2BTINYnSxZYXAdM5wkH/ys9guyNc05hpUjepchL/q7mpBB2XorofMbDA3oas5SvnJqRRb9oWy3Har1rtb58GeiRrhLCZLqJk7cZmCYNnlHsXt3GLRt3kkDYLe2X4enBcuJcuhYUxub1b8U6Pz0j13sIMRhYmtw5IwwOnm1VXQKPc9lRXNBG9lOXQi8AcCDERKNg="
-    skip_cleanup: true
-    on:
-      repo: buildit/gravity-ui-sass
-      tags: true
-      condition: "$TRAVIS_TAG =~ $NPM_TAG_REGEXP"
+jobs:
```

Comment:
I _think_ this part can be simplified. If I'm reading the Travis docs correctly, we don't need to use [stages](https://docs.travis-ci.com/user/build-stages/) at all. They're for running multiple jobs concurrently, but for our current needs just the single (default) job suffices.

I'd argue that running `npx semantic-release` is just another step in the existing `deploy` phase. So I think the `provider: script` and subsequent lines can be moved up to the same place where the old NPM deploy stuff was which you've now removed.